App.js

```diff
@@ -22,19 +22,34 @@ export default class App extends Component<{}> {
     usabilla.setFormDidFailLoading((reminder) => console.log("Error loading form:", reminder))
     usabilla.setFormDidClose((reminder) => console.log(reminder.formId))
   }
+
   onFormLoaded() {
     usabilla.showLoadedForm()
   }
+
   requestForm() {
     usabilla.loadFeedbackForm("YOUR_FORM_ID_HERE")
   }
+
+  takeScreenshotFromActivity() {
+    usabilla.takeScreenshotFromActivity()
+  }
+
+  takeScreenshotFromView(e) {
```

Comment:
what is e?

Reply:
The `e` in this case it a reference to the UI element that called this function... it may not be the most clear name or way to do it, I can change the name while researching for another option